clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -257,6 +263,8 @@ export default async function handler(
   /* eslint-disable @typescript-eslint/no-use-before-define */
   const customFidesCss = await fetchCustomFidesCss(req);
 
+  const { initialize: initializeQuery } = req.query;
```

Comment:
nit: Id prefer to colocated the little bit of logic that checks for the value "false" here and comment why this is being done, instead of putting that online below with the script string building. It feels easier to come back to this later.